src/snowcli/cli/streamlit/manager.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import annotations
+
+import logging
+import typer
+from pathlib import Path
+from typing import List, Optional, Tuple
+
+from snowflake.connector.cursor import SnowflakeCursor
+
+from snowcli.cli.common.sql_execution import SqlExecutionMixin
+from snowcli.cli.snowpark_shared import snowpark_package
+from snowcli.cli.stage.manager import StageManager
+from snowcli.utils import (
+    generate_streamlit_environment_file,
+    generate_streamlit_package_wrapper,
+)
+
+log = logging.getLogger(__name__)
+
+
+class StreamlitManager(SqlExecutionMixin):
+    @staticmethod
+    def get_standard_stage_name(name: str) -> str:
+        # Handle embedded stages
+        if name.startswith("snow://"):
+            return name
+
+        return f"@{name}"
+
+    def list(self) -> SnowflakeCursor:
+        return self._execute_query("show streamlits")
+
+    def description(
+        self, streamlit_name: str
+    ) -> Tuple[SnowflakeCursor, SnowflakeCursor]:
+        description = self._execute_query(f"describe streamlit {streamlit_name}")
+        url = self._execute_query(
+            f"call SYSTEM$GENERATE_STREAMLIT_URL_FROM_NAME('{streamlit_name}')"
+        )
+        return description, url
+
+    def create(
+        self,
+        streamlit_name: str,
+        file: Path,
+        from_stage: str,
+        use_packaging_workaround: bool,
+    ) -> SnowflakeCursor:
+        connection_ctx = self._conn.ctx
+        if from_stage:
+            if "." in from_stage:
+                full_stage_name = from_stage
+            else:
+                full_stage_name = (
+                    f"{connection_ctx.database}.{connection_ctx.schema}.{from_stage}"
+                )
```

Comment:
I don't think this is neccessary, the connection will use the context database by default. So we can simply use `from_stage` as `stage_name`. WDYT?

Reply:
I agree, removed